src/java/org/apache/cassandra/db/Keyspace.java

```diff
@@ -309,8 +311,11 @@ public boolean snapshotExists(String snapshotName)
      */
     public static void clearSnapshot(String snapshotName, String keyspace)
     {
+        RateLimiter snapshotRateLimiter = RateLimiter.create(DatabaseDescriptor.getSnapshotLinksPerSecond() == 0 ?
```

Comment:
the local variable name can change from snapshotRateLimiter to clearSnapshotRateLimiter, though they share the same value but they are of different use.

Reply:
This is addressed.